contracts/abstract/ReaperBaseStrategy.sol

```diff
@@ -50,12 +50,9 @@ abstract contract ReaperBaseStrategy is
      * {MAX_FEE} - Maximum fee allowed by the strategy. Hard-capped at 10%.
      * {STRATEGIST_MAX_FEE} - Maximum strategist fee allowed by the strategy (as % of treasury fee).
      *                        Hard-capped at 50%
-     * {MAX_SECURITY_FEE} - Maximum security fee charged on withdrawal to prevent
-     *                      flash deposit/harvest attacks.
      */
     uint256 public constant MAX_FEE = 1000;
     uint256 public constant STRATEGIST_MAX_FEE = 5000;
-    uint256 public constant MAX_SECURITY_FEE = 10;
```

Comment:
There is a max cap already so it is not like it could be set to anything. 10 is a very low number.